config/antlr.m4

```diff
@@ -20,6 +20,10 @@ AC_DEFUN([AC_PROG_ANTLR], [
       AC_MSG_RESULT([OK])
     fi
   fi
+  # Check if antlr-3.4 was installed via contrib/get-antlr3.4
+  if test -e "../../../antlr-3.4/bin/antlr3"; then
```

Comment:
$ac_abs_top_srcdir is empty, $srcdir would be ../../../ and $ac_pwd would be the absolute path to the build directory. Didn't find any other variables that would be more suitable. Any suggestions?